packages/shadow/src/index.js

```diff
@@ -0,0 +1,16 @@
+// @styled-system/shadow
+import { system } from '@styled-system/core'
+
+export const boxShadow = system({
+  boxShadow: {
+    property: 'boxShadow',
+    scale: 'shadows',
+  },
+})
+
+export const textShadow = system({
+  textShadow: {
+    property: 'textShadow',
+    scale: 'shadows',
+  },
+})
```

Comment:
Thanks for working on this! For the v5 packages, I've been grouping all the utilities into categories so that one import/function will add several style props. E.g. with `typography`, it adds `fontFamily`, `fontSize`, `lineHeight`, etc. as props. I think this `shadow` package could follow the same format, where one `shadow` function adds both `boxShadow` and `textShadow` props. To do the same here, it'd mean combining the two `system` calls into a single one and merging the two configuration options

Reply:
Going to merge this as-is and combine these in the v5 branch